lib/core/src/support/Utils.ts

```diff
@@ -19,7 +19,14 @@ export class Utils {
     const chrome: boolean = !!(window as any).chrome
       && (!!(window as any).chrome.webstore || !!(window as any).chrome.runtime)
 
+    const edge: boolean = chrome && (navigator.userAgent.indexOf("Edg") != -1)
+
+    const opera: boolean = (!!(window as any).opr && !!(window as any).addons)
```

Comment:
If you used any external resources for this assertion I'd add a comment with a link. Should this break, there's a good chance that the answer will be updated in the link too.

Reply:
thanks Michael good idea 👍